.gitignore

```diff
@@ -0,0 +1,4 @@
+**/bazel-bin/
```

Comment:
I believe we don't normally check in `.gitignore`.

Reply:
Removed